components/camel-aws/camel-aws2-sqs/src/main/java/org/apache/camel/component/aws2/sqs/Sqs2Consumer.java

```diff
@@ -375,16 +371,6 @@ protected void doStart() throws Exception {
         }
 
         super.doStart();
-
-        // health-check is optional so discover and resolve
-        healthCheckRepository = HealthCheckHelper.getHealthCheckRepository(getEndpoint().getCamelContext(), "components",
-                WritableHealthCheckRepository.class);
-
-        if (healthCheckRepository != null) {
-            consumerHealthCheck = new Sqs2ConsumerHealthCheck(this, getRouteId());
-            consumerHealthCheck.setEnabled(getEndpoint().getComponent().isHealthCheckEnabled() && getEndpoint().getComponent().isHealthCheckConsumerEnabled());
```

Comment:
This solution does not honor these configuration options. Should they be removed, or should I override `doBuild()` in order to only add the health check in case it is configured?

Reply:
Ah these new options are added in v4 - I am moving this to the scheduled poll consumer, so in your PR you can just remove these lines.

Reply:
https://github.com/apache/camel/pull/10849